components/mdc/Menu/Menu.svelte

```diff
@@ -0,0 +1,63 @@
+<!-- https://github.com/material-components/material-components-web/tree/master/packages/mdc-menu -->
+<script>
+import { onDestroy, onMount } from 'svelte'
+import { MDCMenu } from '@material/menu'
+import { createEventDispatcher } from 'svelte'
+import { goto } from '@roxi/routify';
+
+export let menuItems = []
+export let menuOpen = false
+
+let menu = {}
+let element = {}
+
+const dispatch = createEventDispatcher()
+
+$: currentUrl = window.location.pathname
+$: menu.open = menuOpen
+
+onMount(() => {
+  menu = new MDCMenu(element)
+
+  menu.setDefaultFocusState() //TODO figure out how to use this method and set focus
+
+  return () => menu.destroy()
+})
+
+onDestroy(() => {
+  dispatch('syncToggler')
+})
```

Comment:
this event never fires so we should get rid of it and the onDestroy import. It is ok though because the <svelte:body on:click> closes it.